HTTP_HEADER_FORMAT.md

```diff
@@ -10,49 +10,48 @@ balancers, etc.)
 
 ## Header name
 
-`Trace-Context`
+`Trace-Context` is the name of the [Http header field](https://en.wikipedia.org/wiki/List_of_HTTP_header_fields) being defined here. 
 
 ## Field value
 
-`base16(<version>)-<version_format>`
+The format of the value a list of data values, each value is identified by a 
+unique prefix character the defines the meaning of the value as well as its
+format.  
 
-The value will be US-ASCII encoded (which is UTF-8 compliant). Character `-` is
-used as a delimiter between fields.
+values of a particular kind need not necessarily be present, but they must be 
+in the order defined below (e.g. if the Trace-id is present it must be first)
 
-### Version
+The prefix values currently defined are
 
-Is a 1-byte representing a 8-bit unsigned integer. Version 255 reserved.
+### Prefix '*' The Trace-id 
 
-### Version = 0
+The '*' character indicates that the next 32 characters are hexadecimal digits (lower case)
+that represent a 16 byte identifier.  This identifier is meant to uniquely identify 
+new 'trace'.   The intent is that all requests that are caused directly or 
```

Comment:
if traceID is always the first element in the header string, what's the role of the prefix? If it specifically indicates "hexadecimal string", then the current Spec doesn't allow anything else anyway (#16).

Reply:
You are correct that we may wish to simply drop the  '*' prefix  (I suggested that as an option in the original submission comment).    The value it does have is that it allows us to have Trace-Contexts that encode a Trace-id (or differently (e.g. Base64), some other string etc).     

It is a worthy point of discussion.